src/main/resources/assets/structurize/lang/en_us.json

```diff
@@ -54,6 +54,8 @@
     "item.possetter.anchorpos": "Anchor pos set to x=%d y=%d z=%d",
     "item.possetter.firstpos": "First pos set to x=%d y=%d z=%d",
     "item.possetter.secondpos": "Second pos set to x=%d y=%d z=%d",
+    "item.possetter.missingpos1": "Missing first position",
+    "item.possetter.missingpos2": "Missing final position",
```

Comment:
should this be second instead of final?

Reply:
Fair.  The NBT tags are named Start/end, but that's a good deal clearer.